app/scripts/lib/ComposableObservableStore.js

```diff
@@ -40,7 +40,9 @@ class ComposableObservableStore extends ObservableStore {
   getFlatState () {
     let flatState = {}
     for (const key in this.config) {
-      flatState = { ...flatState, ...this.config[key].getState() }
+      const child = this.config[key]
+      const childState = child.getState ? child.getState() : child.state
```

Comment:
These stopgap changes are so this composable store can support the old `obs-store` API as well as the new `gaba/BaseController` API. Once all child controllers and the MetaMask controller are converted, this module won't be used and will go away entirely.